### DISCUSSION TITLE
Using different models / Unable to run any other model except ggml-gpt4all-j-v1.3-groovy.bin

### DISCUSSION THREAD

**Discussion by Jaskirat3690** [imartinez/privateGPT#697] [Q&A]
Hey all! I have been struggling to try to run privateGPT.py on any other models. I have tried 4 models: ggml-gpt4all-l13b-snoozy.bin, ggml-v3-13b-hermes-q5_1.bin, ggml-mpt-7b-instruct.bin and Manticore-13B.ggmlv3.q8_0 (all downloaded from gpt4all website).
I have tried hanging the model type to GPT4All and LlamaCpp, but I keep getting different errors. 

%%%%%  Here is my cmd when running the ggml-gpt4all-l13b-snoozy.bin model using LlamaCpp model_type :

D:\PrivateGPT\privateGPT-main>python privateGPT.py
Using embedded DuckDB with persistence: data will be stored in: db
llama.cpp: loading model from models/ggml-gpt4all-l13b-snoozy.bin
llama_model_load_internal: format     = ggjt v1 (pre #1405)
llama_model_load_internal: n_vocab    = 32000
llama_model_load_internal: n_ctx      = 1000
llama_model_load_internal: n_embd     = 5120
llama_model_load_internal: n_mult     = 256
llama_model_load_internal: n_head     = 40
llama_model_load_internal: n_layer    = 40
llama_model_load_internal: n_rot      = 128
llama_model_load_internal: ftype      = 2 (mostly Q4_0)
llama_model_load_internal: n_ff       = 13824
llama_model_load_internal: n_parts    = 1
llama_model_load_internal: model size = 13B
error loading model: this format is no longer supported (see https://github.com/ggerganov/llama.cpp/pull/1305)
llama_init_from_file: failed to load model
Traceback (most recent call last):
  File "D:\PrivateGPT\privateGPT-main\privateGPT.py", line 76, in <module>
    main()
  File "D:\PrivateGPT\privateGPT-main\privateGPT.py", line 34, in main
    llm = LlamaCpp(model_path=model_path, n_ctx=model_n_ctx, callbacks=callbacks, verbose=False)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "pydantic\main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for LlamaCpp
__root__
  Could not load Llama model from path: models/ggml-gpt4all-l13b-snoozy.bin. Received error  (type=value_error)

%%%%% Here is my cmd when running the ggml-gpt4all-l13b-snoozy.bin model using GPT4All model_type :

D:\PrivateGPT\privateGPT-main>python privateGPT.py
Using embedded DuckDB with persistence: data will be stored in: db
Found model file.
gptj_model_load: loading model from 'models/ggml-gpt4all-l13b-snoozy.bin' - please wait ...
gptj_model_load: invalid model file 'models/ggml-gpt4all-l13b-snoozy.bin' (bad magic)
GPT-J ERROR: failed to load model from models/ggml-gpt4all-l13b-snoozy.bin
Enter a query: exit

%%%%% Here is my cmd correctly running the groovy model:

D:\PrivateGPT\privateGPT-main>python privateGPT.py
Using embedded DuckDB with persistence: data will be stored in: db
Found model file.
gptj_model_load: loading model from 'models/ggml-gpt4all-j-v1.3-groovy.bin' - please wait ...
gptj_model_load: n_vocab = 50400
gptj_model_load: n_ctx   = 2048
gptj_model_load: n_embd  = 4096
gptj_model_load: n_head  = 16
gptj_model_load: n_layer = 28
gptj_model_load: n_rot   = 64
gptj_model_load: f16     = 2
gptj_model_load: ggml ctx size = 5401.45 MB
gptj_model_load: kv self size  =  896.00 MB
gptj_model_load: ................................... done
gptj_model_load: model size =  3609.38 MB / num tensors = 285

Enter a query:

%%%%%%
I get similar errors when running the other models. Only groovy model is working. Any help is appreciated. Thanks.

**Comment by 77bgh** [comment]
see this topic

https://github.com/imartinez/privateGPT/discussions/580
